### ISSUE TITLE
API: Supersede /raw/v1 with /rest/v1

### ISSUE THREAD

**Issue by equill** [equill/restagraph#28]
The current RAW API is REST-inspired, but in practice is a hybrid of REST and GRAPHQL.

Rather than create `/raw/v2` with correct REST syntax and semantics, supersede it with an actual REST API, including HATEOAS. Then, once that's complete, it sets the stage for implementing `/graphql/v1`.

This also means RG's (and hence Syscat's) release and promotion doesn't have to be blocked on rectification of `/raw/v1`.

**Comment by equill** [comment]
Won't do.

After comparing RG's API with both REST and GraphQL, RG is neither of these. They're both implicitly based around RDBMSes, while RG is explicitly graph-oriented.
